css/customstyles.css

```diff
@@ -1284,16 +1255,18 @@ div.box.box1 {
 
 .highlight-box {
   font-size: 11pt;
-  border: 1px solid #cccccc;
-  background-color: #EBF5EB;
+  border: 1px solid #dadfff;
+  background-color: #f3f5ff;
   padding: 10px;
   border-radius: 3px;
-  margin: 10px auto 20px auto;
-  width: 680px;
+  margin: 10px auto;
   padding: 10px;
 }
 
-pre.highlight {margin-top: 0;}
+pre.highlight {
+    margin-top: 0;
+    padding: 16px;
+}
```

Comment:
if i add `color: white;` to `pre.highlight`, the code snippet's text will become white again.